pkg/syncer/mysqlcluster/settings.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2018 Pressinfra SRL
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysqlcluster
+
+import (
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	// MysqlPortName represents the mysql port name.
+	MysqlPortName = "mysql"
+	// MysqlPort is the default mysql port.
+	MysqlPort = 3306
+
+	// HelperXtrabackupPortName is name of the port on which we take backups
+	HelperXtrabackupPortName = "xtrabackup"
+	// HelperXtrabackupPort is the port on which we serve backups
+	HelperXtrabackupPort = 3307
+
+	// OrcTopologyDir path where orc conf secret is mounted
+	OrcTopologyDir = "/var/run/orc-topology"
+
+	rStrLen = 18
```

Comment:
What is `rStrLen`? This can be moved into `pkg/syncer/mysqlcluster/secret.go` since it's used only there.